--- wvstreams-4.6.1/crypto/wvx509.cc	2008-10-23 22:23:49.000000000 +0200
+++ wvstreams-4.6.1/crypto/wvx509.cc	2013-03-17 12:40:33.207445706 +0100
@@ -1157,7 +1157,7 @@
         
         if (ext)
         {
-            X509V3_EXT_METHOD *method = X509V3_EXT_get(ext);
+            const X509V3_EXT_METHOD *method = X509V3_EXT_get(ext);
             if (!method)
             {
                 WvDynBuf buf;
