[PATCH] remove doxygen from install target
Index: libroxml-2.1.1/Makefile
===================================================================
--- libroxml-2.1.1.orig/Makefile
+++ libroxml-2.1.1/Makefile
@@ -140,7 +140,7 @@
 	$E - $(MAKE) -C $(abspath fuse.xml)
 
 .PHONY: install
-install: $(TARGETS) doxy
+install: $(TARGETS)
 	$P '  INSTALL DIRS'
 	$E mkdir -p $(DESTDIR)/usr/bin
 	$E mkdir -p $(DESTDIR)/usr/include
