Index: libroxml-2.1.0/Makefile
===================================================================
--- libroxml-2.1.0.orig/Makefile
+++ libroxml-2.1.0/Makefile
@@ -113,11 +113,11 @@
 .PHONY : doxy
 doxy : doxy.cfg man.cfg
 	$P '  DOXYGEN'
-	$E - doxygen doxy.cfg &>/dev/null
+	$E - doxygen doxy.cfg
 	$E - cp data/icons/roxml.png docs/html/
 	$E - cp data/icons/libroxml-ex.png docs/html/
 	$P '  MAN'
-	$E - doxygen man.cfg &>/dev/null
+	$E - doxygen man.cfg
 	$E - chmod -R a+rw docs
 
 .PHONY: clean
